app-android-xamarin/Moodis/Moodis/Feature/CameraFeature/CameraFragment.cs

```diff
@@ -38,12 +47,46 @@ public override View OnCreateView(LayoutInflater inflater, ViewGroup container,
             return view;
         }
 
+        public override async void OnViewCreated(View view, Bundle savedInstanceState)
```

Comment:
Don't think this needs async as you only await on line 65 which is already handled by AfterTakenPicture = async (sender, e) =>